libs/styles/breakpoint-theme.css

```
@@ -0,0 +1,193 @@
+
+  
+/* Static Links */
+.con-block.static-links[class*="dark-"] a:not(.con-button),
+.static-links .con-block[class*="dark-"] a:not(.con-button),
+.con-block.static-links[class*="light-"] a:not(.con-button),
+.static-links .con-block[class*="light-"] a:not(.con-button),
+.con-block.static-links[class*="dark-"] a:not(.con-button):hover,
+.static-links .con-block[class*="dark-"] a:not(.con-button):hover,
+.con-block.static-links[class*="light-"] a:not(.con-button):hover,
+.static-links .con-block[class*="light-"] a:not(.con-button):hover {
+  color: inherit;
+}
+
+/* mobile ONLY */
+@media (max-width: 600px) {
+  /* mobile-only theme */
+  .con-block.light-mobile-only { color: var(--text-color); }
+  .con-block.dark-mobile-only {  color: var(--color-white); }
```

Nit of the year: double spacing before `color` 😅 Same for tablet and desktop
```suggestion
  .con-block.dark-mobile-only { color: var(--color-white); }
```

---

Ill check my linting, This should have been flagged. These are resolved now.